docs/client/iOS/_options.mdx

```diff
@@ -11,6 +11,43 @@
    - StatsigEnvironment is a class for you to set environment variables that apply to all of your users in the same session and will be used for targeting purposes.
    - e.g. passing in a value of `StatsigEnvironment(tier: .Staging)` will allow your users to pass any condition that pass for the staging environment tier, and fail any condition that only passes for other environment tiers.
 
+- **evaluationCallback**: EvaluationCallbackData, default `nil`
+  - EvaluationCallback provides a callback when an evaluation is made against one of your configurations
+  (gate, dynamic config, experiment, layer config and parameter stores). This is useful when you want to trigger specific actions or log evaluations based on the results received from Statsig.
```

Comment:
"layer config" is not a term. We have "layer_configs" but that is purely internal. Just say "layer"